luatest/server.lua

```diff
@@ -59,7 +59,7 @@ end
 
 --- Build a server object.
 --
--- @tab object
+-- @tab[opt] object
```

Comment:
The 'object' is not explained. Is it some prototype object to base the new server object on?

Reply:
Fixed.